### DISCUSSION TITLE
Is 8.10.2 bug fix release imminent - should I hold off distributing 8.10.1?

### DISCUSSION THREAD

**Discussion by BrianInglis** [curl/curl#14994] [General]
I noticed RELEASE-NOTES being synced. 
Is an 8.10.2 regression bug fix release imminent? 
Should I hold off distributing 8.10.1?

**Comment by dfandrich** [comment]
The feature window just opened, which means there won't be another 8.10 point
release: https://curl.se/mail/lib-2024-09/0021.html

**Comment by BrianInglis** [comment]
Thanks Dan, Wondered when I saw `8.10.2` appear in `RELEASE-NOTES` while waiting for `8.10.1` to build and run tests for Cygwin release. I'll upload an announce it for distribution. Take care. Thanks, Brian

**Comment by bagder** [comment]
We just haven't updated the file in git to say `8.11.0` yet. It will be done soon. The really important version number is however what it says in releases.
